docs/synctl-get-result.md

```diff
@@ -36,3 +37,14 @@ Save HAR file to local
 ```
 synctl get result <id> --test <test-id> --har
 ```
+
+Show all CI-CD test results
+```
+synctl get result --ci-cd
+```
+
+Show a CI-CD test result
+```
+synctl get result <id> --ci-cd
+
+```
```

Comment:
Issue: These two are not to get ci-cd test results, they are for getting ci-cd tests and getting one ci-cd test based on a given test result id.